### ISSUE TITLE
Variables in civicrm_setting should have their own namespace

### ISSUE THREAD

**Issue by alexymik** [progressivetech/net.ourpowerbase.sumfields#100]
Currently the variables for Summary Fields are stored in the `civicrm_setting` table using their "local" names:

* `generate_schema_and_data`
* `event_type_ids`

I would suggest to namespace these to something like `sumfields_`, ex:

* `sumfields_generate_schema_and_data`
* `sumfields_event_type_ids`

This would make it more clear what the variables are when viewed outside the Summary Field context.

`sumfields_get_setting()` could be modified to directly use this as a prefix so most of the code can remain the same.

**Comment by jmcclelland** [comment]
Yes - that is a good idea. Would you be up for a PR? I think it would also need an upgrade step to rename the existing settings.
